site/content/docs/integrations/prometheus-v2.md

```diff
@@ -125,7 +125,7 @@ To calculate the percentage of check runs that failed in the last 24 hours, use:
 increase(checkly_check_result_total{status="failing"}[24h]) / ignoring(status) sum without (status) (increase(checkly_check_result_total[24h]))
 ```
 
-The `checkly_check_result_total` counter is reset to `0` every hour. The [`increase`](https://prometheus.io/docs/prometheus/latest/querying/functions/#increase) function will handle these resets automatically, though. `ignoring(status)` is needed so that Prometheus can perform [vector matching](https://prometheus.io/docs/prometheus/latest/querying/operators/#vector-matching) on the division operation. 
+The `checkly_check_result_total` counter is reset to `0` every hour. Prometheus handles resets automatically, so it is possible to use all functions like [`increase`](https://prometheus.io/docs/prometheus/latest/querying/functions/#increase) or [`rate`](https://prometheus.io/docs/prometheus/latest/querying/functions/#rate) for these series.
```

Comment:
I think we could rephrase it a bit more - user's don't have the option to use `increase` or `rate`, they need to use `increase`/`rate`. Also should we still mention `ignoring(status)`? It seems like a strange part of the query and it's good to explain it IMO.

I think the initial phrasing is OK? We also have another mention of `increase` and `rate` here:
![image](https://github.com/user-attachments/assets/40939728-f04d-45c8-9533-1ebee49f9fe7)

Reply:
then let's simplify even more and only do
```
The `checkly_check_result_total` counter is reset to `0` every hour. Prometheus handles resets automatically, so it is possible to use all functions.
```
?